common/src/main/java/com/wynntils/utils/MathUtils.java

```diff
@@ -59,4 +59,15 @@ public static int integerFromRoman(String numeral) {
         }
         return num;
     }
+
+    public static double map(double sourceNumber, double fromA, double fromB, double toA, double toB) {
+        double deltaA = fromB - fromA;
+        double deltaB = toB - toA;
+        double scale = deltaB / deltaA;
+        double negA = -1 * fromA;
+        double offset = (negA * scale) + toA;
+        double finalNumber = (sourceNumber * scale) + offset;
+        int calcScale = (int) Math.pow(10, 5);
+        return (double) Math.round(finalNumber * calcScale) / calcScale;
```

Comment:
This calcScale thing. 🤔 My first thought was that you wanted to limit the result to a certain number of significant (decimal) digits. But is that really what you are doing? It seems to me that it does nothing at all for large numbers, and risk making very small numbers all 0.